.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Pack and publish NuGet packages
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install dotnet7
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 7.0.x
```

Comment:
Any reason we are using .NET 7 instead of 8 ? The library we are building is .netstandard 2.1

Reply:
It was recommended in the docs. Therefore I choose to keep it that way 🙂 But you are correct it should be .NET 8 👍

Reply:
Docs might have been created before .net 8 was release. This would typically be one of the discrepancies in the docs that you should update as a part of this issue, @codemascot

Reply:
Unresolving so you can resolve once pr for docs has been created as well

Reply:
Here is the `altinn-studio-docs` PR: https://github.com/Altinn/altinn-studio-docs/pull/1402

Resolving the issue now.